src/Domain/Model/DynamoDbModel.php

```diff
@@ -183,6 +183,10 @@ public function save(array $options = [])
             $this->fireModelEvent('creating');
         }
 
+        if ($this->usesTimestamps()) {
```

Comment:
Ah, OK, then this is alright. I'll have a look at this at some point, and merge when I have time.